trawler/src/main/java/io/m9sweeper/trawler/scanners/Trivy.java

```diff
@@ -117,6 +117,18 @@ public void runScan() throws Exception {
 
             // Export the location of this file so that Trivy can utilize it
             trivyScanCommandBuilder.append("export GOOGLE_APPLICATION_CREDENTIALS='").append(gcrAuthFile.getAbsolutePath()).append("'; ");
+        } else if (registry.getAuthType().equals("AZCR")) {
```

Comment:
not a blocker, but wanted to mention in Java people oftentimes flip it around like this: 


"AZR".equals(registry.getAuthType())

That way if auth type is null/not set, it doesn't throw a null pointer exception

Reply:
See also: https://en.wikipedia.org/wiki/Yoda_conditions